src/swaps/bitcoinv2.rs

```diff
@@ -888,8 +1015,9 @@ impl BtcSwapTxV2 {
     /// Multiply the size by the fee_rate to get the absolute fees.
     pub fn size(&self, keys: &Keypair, preimage: &Preimage) -> Result<usize, Error> {
         let dummy_abs_fee = 5_000;
+        // Can only calculate non-coperative claims
```

Comment:
In case there is a need for calculating those, schnorr signatures are always 64 bytes. Could be stubbed